<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1236</width>
      <height>400</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Groups">
     <attribute name="title">
      <string>Groups</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>580</width>
        <height>300</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QGroupBox" name="GB_Reporting">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>1</y>
        <width>411</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>Reporting</string>
      </property>
      <widget class="QColumnView" name="CV_RegisteredUsers">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>151</width>
         <height>201</height>
        </rect>
       </property>
       <property name="resizeGripsVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QColumnView" name="CV_ReportRecipients">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>40</y>
         <width>151</width>
         <height>201</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_Add">
       <property name="geometry">
        <rect>
         <x>166</x>
         <y>110</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_Remove">
       <property name="geometry">
        <rect>
         <x>166</x>
         <y>140</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Report Recipients</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Registered Users</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GB_Scheduling">
      <property name="geometry">
       <rect>
        <x>589</x>
        <y>1</y>
        <width>221</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>Scheduling</string>
      </property>
      <widget class="QCheckBox" name="CB_Thursday">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>290</y>
         <width>76</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Thursday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="CB_Tuesday">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>250</y>
         <width>76</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Tuesday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="CB_Sunday">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>210</y>
         <width>76</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Sunday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="CB_Saturday">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>330</y>
         <width>76</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Saturday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="CB_Friday">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>310</y>
         <width>76</width>
         <height>20</height>
        </rect>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Friday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="CB_Wednesday">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>270</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Wednesday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="CB_Monday">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>230</y>
         <width>76</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Monday</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RB_Weekly">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Weekly</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RB_Weekdays">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Weekdays</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RB_Monthly">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter numberic day of month OR 31 for the last day of the month&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Monthly</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RB_Daily">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Daily</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RB_Disabled">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Disabled</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="RB_Custom">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Custom</string>
       </property>
      </widget>
      <widget class="QComboBox" name="DD_WeeklyChosen">
       <property name="geometry">
        <rect>
         <x>109</x>
         <y>136</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxVisibleItems">
        <number>7</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="IB_DayOfMonthChosen">
       <property name="geometry">
        <rect>
         <x>109</x>
         <y>180</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="RB_BiWeekly">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Bi-Weekly</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>16</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:32pt;&quot;&gt;}&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="PB_AddGroup">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Group</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PB_DeleteGroup">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>340</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Group</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>59</x>
        <y>310</y>
        <width>521</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>File Name:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Users">
     <attribute name="title">
      <string>Users</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="PB_Close">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>420</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1256</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGroups">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>102</y>
      <width>135</width>
      <height>50</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>Log In</string>
    </property>
   </widget>
   <addaction name="menuGroups"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
